explorer/client/src/utils/stringUtils.ts

```diff
@@ -48,6 +50,28 @@ export function truncate(fullStr: string, strLen: number, separator?: string) {
     );
 }
 
+export async function extractFileType(displayString: string) {
+    const OPTIONS = ['PNG', 'JPG', 'JPEG'];
+    // First check file extension:
+    let result = displayString.split('.').at(-1)?.toUpperCase();
```

Comment:
Is there a reason we even inspect the file extension given it could be wrong (e.g. png extension could serve non-png content-type)?

Reply:
This has been removed.

Reply:
Turns out we will need the file extension checker in some cases, specifically when CORS stops access to the Content-Type header (see below example). The approach taken is to first check the the header and then the file extension.

![image](https://user-images.githubusercontent.com/11377188/182685322-13e3c35d-7598-4b05-bd5e-3c6a5fa0785d.png)